renderer/brave_url_loader_throttle_provider_impl.h

```diff
@@ -0,0 +1,38 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_RENDERER_BRAVE_URL_LOADER_THROTTLE_PROVIDER_IMPL_H_
+#define BRAVE_RENDERER_BRAVE_URL_LOADER_THROTTLE_PROVIDER_IMPL_H_
+
+#include <memory>
+
+#include "base/functional/callback.h"
+#include "chrome/renderer/url_loader_throttle_provider_impl.h"
+
+class BraveURLLoaderThrottleProviderImpl
+    : public URLLoaderThrottleProviderImpl {
+ public:
+  BraveURLLoaderThrottleProviderImpl(
+      blink::ThreadSafeBrowserInterfaceBrokerProxy* broker,
+      blink::URLLoaderThrottleProviderType type,
+      ChromeContentRendererClient* chrome_content_renderer_client,
```

Comment:
I think can you pass `BraveContentRendererClient` here, save it as a member and access `is_tor` via a class function.

Reply:
fixed in f5a95cb28ea818c4d7c59df9221985c011713e54